app/views/layouts/application.html.erb

```diff
@@ -7,9 +7,25 @@
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
   </head>
 
   <body>
+    <!-- insert navbar -->
+    <nav id='fixed-nav-bar'>
+      <div class="">
+        <ul style="display: inline-block;">
```

Comment:
cant you extract `display: inline` & `display: inline-block;` to a css class?

Reply:
Done